dora/underfs/oss/src/main/java/alluxio/underfs/oss/OSSUnderFileSystem.java

```diff
@@ -380,9 +382,77 @@ public static ClientBuilderConfiguration initializeOSSClientConfig(
     ossClientConf.setConnectionTTL(alluxioConf.getMs(PropertyKey.UNDERFS_OSS_CONNECT_TTL));
     ossClientConf.setMaxConnections(alluxioConf.getInt(PropertyKey.UNDERFS_OSS_CONNECT_MAX));
     ossClientConf.setMaxErrorRetry(alluxioConf.getInt(PropertyKey.UNDERFS_OSS_RETRY_MAX));
+    if (isProxyEnabled(alluxioConf)) {
+      ossClientConf.setProxyHost(getProxyHost(alluxioConf));
+      ossClientConf.setProxyPort(getProxyPort(alluxioConf));
+      ossClientConf.setProtocol(getProtocol());
```

Comment:
add a log here if proxy is used (so you know)

Reply:
good idea, done